.github/workflows/stale.yml

```diff
@@ -5,9 +5,6 @@ on:
     # check issue and pull request once at 01:30 a.m. every day
     - cron: '30 1 * * *'
 
-permissions:
-  contents: read
```

Comment:
Can we use 'write' explicitly to avoid changes by mistake?

Reply:
yes